.release/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "dummy-release",
+  "version": "1.0.0",
+  "description": "just a dummy to run the release",
+  "main": "index.js",
+  "directories": {
+    "example": "../examples"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Hapag-Lloyd/log4j2-filtered-stacktrace-plugin.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Hapag-Lloyd/log4j2-filtered-stacktrace-plugin/issues"
+  },
+  "homepage": "https://github.com/Hapag-Lloyd/log4j2-filtered-stacktrace-plugin#readme",
+  "dependencies": {
+    "@semantic-release/changelog": "^6.0.3",
+    "@semantic-release/commit-analyzer": "^11.0.0",
+    "@semantic-release/exec": "^6.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^9.0.0",
+    "@semantic-release/release-notes-generator": "^12.0.0",
+    "conventional-changelog-conventionalcommits": "^7.0.2",
+    "semantic-release": "^22.0.0"
```

Comment:
There are some new major versions in the `semantic-release` ecosystem. We should use the latest major version here to keep renovate PRs at a minimum

Reply:
I usually let Renovate do the nasty work ;-)